tests/bootstrap.php

```diff
@@ -133,6 +133,7 @@
 
 ini_set('intl.default_locale', 'en_US');
 ini_set('session.gc_divisor', '1');
+ini_set('assert.exception', '1');
```

Comment:
I feel like this is something we should add to the application skeleton too.

Reply:
Wonder if we should check for this setting in some sort of debug mode/path.

Reply:
Currently I have set this value in `ExceptionTrap::register()` (which is called from app's bootstrap) since I want exceptions to be thrown in case of assertion failures (setting it to `0` shows only warning).

`zend.assertions` is the ini config which controls whether assertion code is generated and it can only be set through `php.ini` file, not at runtime. As per the manual it's default value is `1` (generate and execute code) but Ubuntu for e.g. seems to sets it to `-1` (do not generate code). So maybe in debug mode we should check it's value and instruct users to turn it on.